src/setup/internal.ts

```diff
@@ -17,5 +17,5 @@ export const setup = {
     user: "fission.name"
   },
 
-  shouldPin: true,
+  shouldPin: false,
```

Comment:
I think we can just remove this now, right?
I'm not 100% sure. There might be a use-case for pinning for users who're providing their own IPFS instance.
... But at the same time I think there's noone who's actually doing that. :smile:

Reply:
Can't hurt to keep it around I guess? In case js-ipfs does turn on GC, like how it's supposed to work 😅